spec/spec_helper.rb

```diff
@@ -1,2 +1,5 @@
 $LOAD_PATH.unshift File.expand_path(File.join(File.dirname(__FILE__), '../lib'))
 $LOAD_PATH.unshift File.expand_path(File.join(File.dirname(__FILE__), '../app'))
+
+require 'capybara/rspec'
+require 'dummy/config/environment'
```

Comment:
The point of `spec_helper` is running unit tests that don't require Rails. For the ones that do, we require `spec_helper_integration` (https://github.com/doorkeeper-gem/doorkeeper/blob/master/spec/spec_helper_integration.rb#L11). Can we change the file that's loaded in teh ones that do require Rails, and leave this skinnier?